src/Localization/Localization/src/ResourceManagerStringLocalizer.cs

```diff
@@ -202,7 +202,7 @@ protected IEnumerable<LocalizedString> GetAllStrings(bool includeParentCultures,
 
             try
             {
-                return culture == null ? _resourceManager.GetString(name) : _resourceManager.GetString(name, culture);
+                return culture == null ? _resourceManager.GetString(name, CultureInfo.CurrentUICulture) : _resourceManager.GetString(name, culture);
```

Comment:
Want to double check this right 😄

Reply:
It is. The analyzer insisted that we call the overload which uses UI Culture by default: https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/Resources/ResourceManager.cs#L602-L607